### ISSUE TITLE
Improve README file

### ISSUE THREAD

**Issue by sawyer1997** [sawyer1997/Giant-Wheel#1]

**Comment by shivamsinha545** [comment]
Hey! I'd like to help you out! If you'll assign me this issue, I'll get right on it!

**Comment by AtharvaMutsaddi** [comment]
Hi [sawyer1997](https://github.com/sawyer1997),

I'm interested in contributing to this project and noticed that you've self-assigned the issue for improving the README file. Is the issue still open for contributions? If so, could you provide more details on what exactly you are looking for?

Thanks!
[AtharvaMutsaddi]

**Comment by sawyer1997** [comment]
Yes please, go ahead. I haven't run the project from quite some time. So, I want you to improve the README in a way, that it should be easy to clone the project and run the project very easily. Also, if you can paste some images and instructions on how to run after installation of dependencies, it would be quite helpful. Sorry for replying little late. Let me know if you need any more information on this. Good Luck!
